docs/dev/community.html

```diff
@@ -0,0 +1,223 @@
+                        <li class="active"><a href="http://lfe.io/">LFE</a></li>
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Docs <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li><a href="/current/index.html">Current</a></li>
+                                <li><a href="/dev/index.html">Dev</a></li>
+                                <li><a href="/v1.2/index.html">v1.2</a></li>
+                                <li><a href="/v0.10/index.html">v0.10</a></li>
+                                <li><a href="/v0.9/index.html">v0.9</a></li>
+                                <li><a href="/v0.8/index.html">v0.8</a></li>
+                                <li><a href="/v0.7/index.html">v0.7</a></li>
+                            </ul>
+                        </li>
+                        <li><a href="#start">Getting Started</a></li>
+                        <li><a href="#reference">Reference</a></li>
+                        <li><a href="/dev/community.html">Community</a></li>
+                        <li><a href="http://blog.lfe.io/">Blog</a></li>
+                    </ul>
+                </div><!--/.nav-collapse -->
+            </div>
+
+        </nav>
+        
+        
+        <main>
+            
+            
+            <div class="wrapper">
+
+                <section>
+
+                    <h1>LFE Community Resources</h1>
+
+                    <h2>Mail List and Social Media</h2>
+
+                    <p>If you&#39;d like to ask questions or just keep in touch with the project, you can
+                        do so via:</p>
+
+                    <ul>
+                        <li>Our project <a href="https://twitter.com/ErlangLisp">Twitter account</a>
+                            where we chatter about related topics</li>
+                        <li>Our low-volume
+                            <a href="http://groups.google.com/group/lisp-flavoured-erlang">forum/mail list</a>
+                            on Google Groups</li>
+                        <li>We have an IRC channel -- come by and say &quot;hi&quot; on <a href="https://kiwiirc.com/client/irc.freenode.net/#erlang-lisp">#erlang-lisp</a> (Freenode.net)</li>
+                        <li>A <a href="https://plus.google.com/u/1/communities/103919485468949397234">Google+
+                            Community</a></li>
+                    </ul>
+
+                    <h2>Blogs, News, and Wikis</h2>
+
+                    <ul>
+                        <li>The LFE/LFEX <a href="http://plan.lfe.io/">.plan</a> file</li>
+                        <li>An <a href="http://blog.lfe.io/">LFE news, tutorials, and updates blog</a></li>
+                        <li><a href="http://rvirding.blogspot.com/">Robert Virding&#39;s blog</a></li>                     
```

Comment:
You can re-add my blog from here (just LFE-specific posts):
 * http://oubiwann.blogspot.com/search/label/lfe